pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -96,6 +97,23 @@ let
         maintainers = with maintainers; [ mathnerd314 emily ];
       };
     };
```

Comment:
> Tested using a NixOS test in NickCao@2cb96de to check that it renders correctly

I think you might as well add the test to this PR, easier to keep track of that way.

According to [the manual](https://nixos.org/manual/nixpkgs/stable/#ssec-nixos-tests-linking) it should be
```
  passthru.tests = {
    noto-fonts = nixosTests.noto-fonts;
  };
```

Reply:
I cherry-picked the commit into this PR.

@NickCao Are you okay with https://github.com/NixOS/nixpkgs/pull/156342/commits/7e0ff044a40423b33bafbc8046a6e29692200698 being merged?

Reply:
That's ok (sorry for the late response)